src/main/java/core/basesyntax/service/FruitService.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.AbstractOperation;
+import core.basesyntax.model.OperationFactory;
+import core.basesyntax.model.OperationSet;
+import core.basesyntax.model.entities.Fruit;
+import core.basesyntax.service.io.ReportWriter;
+import java.util.List;
+
+public class FruitService extends AbstractService<Fruit> {
```

Comment:
```suggestion
public class FruitServiceImpl implements FruitService {
```

Reply:
The project is designed in such a way that we can have different types of products, and for each type we will need another service, so FruitService as an abstraction does not make any sense

Reply:
ok, then create the generic ProductService interface and make FruitServiceImpl implement it